recipes/mujs/test.c

```diff
@@ -0,0 +1 @@
+
```

Comment:
Is this file supposed to be empty? If so, can you create it as part of the test?

Reply:
Thanks for the recommendation, I updated the test.